student_class.rb

```diff
@@ -0,0 +1,18 @@
+require_relative 'person_class'
+
+class Student < Person
+  attr_accessor :classroom
+
+  def intialize(classroom)
+    @classroom = classroom
+    super(
+      age,
+      name,
+      parent_permission
+    )
+  end
```

Comment:
- [OPTIONAL] Kindly let's add the age, name, and parent_permission as parameters of the student Constructor and make sure to create an instance variable classroom. It might look like this: 

```ruby
  def initialize(age, classroom, name = "Unknown", parent_permission = true)  
    super(age, name, parent_permission)
    @classroom = classroom
  end
```